package.json

```
@@ -7,7 +7,9 @@
     "build": "next build",
     "start": "next start",
     "eslint": "eslint .",
-    "prettify": "prettier --write ."
+    "prettify": "prettier --write .",
+    "stylelint": "stylelint \"**/*.css\"",
+    "stylelint-check": "stylelint-config-prettier-check"
```

```suggestion
    "stylelint-check": "stylelint-config-prettier-check"
```
not sure if we need this line.